.github/workflows/unit-tests.yaml

```diff
@@ -13,30 +13,24 @@ jobs:
     name: Unit tests
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-
-    # Cache
-    - uses: actions/cache@v1
+    - uses: actions/checkout@v3
+    - uses: actions/cache@v3
       with:
         path: ~/.cache/pip
         key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
         restore-keys: |
-
-    # Setup
-    - name: Set up Python 3.7
-      uses: actions/setup-python@v1
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v4
       with:
-        python-version: 3.7
+        python-version: '3.10'
```

Comment:
your version is quoted here but was not quoted before, is that likely to cause any issue?

Reply:
I have encountered an error with python 10 where if it was unquoted and it stripped the trailing zero. I didn't want to take my chances and used the example from official action - https://github.com/actions/setup-python